MentoringHomeWork/HW4/Task3/Sum3/Sum3.cs

```diff
@@ -0,0 +1,72 @@
+using System.Security;
+
+namespace Task3.Sum3;
+
+public class Sum3
+{
+    public void Task3()
+    {
+        var result = 0;
+        Getnumber();
+        CheckSum();
+        return;
+
```

Comment:
Be careful with new lines - minor: delete useless spaces, there should be just one

Reply:
resolved